src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@import "./untils/_variables";
+@import "./untils/_mixins";
+
+@import "./blocks/reset";
+@import "./blocks/calendar";
```

Comment:
Consider renaming 'calendar' to '_calendar', as it's a partial SCSS file and should have an underscore prefix to indicate it's a partial. 📁